client/src/components/geneExpression/infoPanel/common/infoPanelContainer/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { kebabCase } from "lodash";
+import { InfoContainer, InfoWrapper } from "../style";
+import {
+  ErrorInfo,
+  LoadingInfo,
+  NoneSelected,
+  ShowInfo,
+} from "../infoPanelParts";
+import { ExtendedInfoProps } from "../types";
+
+function ContainerInfo(props: ExtendedInfoProps) {
+  const {
+    id,
+    name,
+    symbol,
+    description,
+    synonyms,
+    references,
+    error,
+    loading,
+    infoType,
+    url,
+    showWarningBanner = false,
+  } = props;
+
+  const infoTypeTag = kebabCase(infoType);
+
+  const wrapperTestId = `${
+    infoType === "Gene" ? symbol : name
+  }:${infoTypeTag}-info-wrapper`;
+
+  return (
+    <InfoWrapper id={`${infoTypeTag}info_wrapper`} data-testid={wrapperTestId}>
+      <InfoContainer id={`${infoTypeTag}-info`}>
+        {/* Loading */}
+        {(name || symbol) && !error && loading && (
+          <LoadingInfo name={name} infoType={infoType} />
+        )}
+
+        {/* None Selected */}
+        {name === "" && error === null && !loading && (
+          <NoneSelected infoType={infoType} />
+        )}
+
+        {/* Error */}
+        {error && <ErrorInfo name={name} infoType={infoType} />}
```

Comment:
I noticed that when I turn off my internet and click on a cell info, the panel stays as Loading instead of showing the error. Is that expected?

Steps:
1. Load the app
2. Turn off internet in console network tab
3. Expand cell categories
4. Find a cell type info button and click
5. Observe that lower right corner shows "Loading..."

Reply:
Great callout! 

I've added some error handling to `fetchCellTypeInfo` - it will now display the error component